src/core/drive/head_snapshot.ts

```diff
@@ -1,71 +1,37 @@
 import { Snapshot } from "../snapshot"
 
-type ElementDetailMap = { [outerHTML: string]: ElementDetails }
-
-type ElementDetails = {
-  type?: ElementType
-  tracked: boolean
-  elements: Element[]
-}
-
-type ElementType = "script" | "stylesheet"
-
 export class HeadSnapshot extends Snapshot<HTMLHeadElement> {
-  readonly detailsByOuterHTML = this.children
-    .filter((element) => !elementIsNoscript(element))
-    .map((element) => elementWithoutNonce(element))
-    .reduce((result, element) => {
-      const { outerHTML } = element
-      const details: ElementDetails =
-        outerHTML in result
-          ? result[outerHTML]
-          : {
-              type: elementType(element),
-              tracked: elementIsTracked(element),
-              elements: [],
-            }
-      return {
-        ...result,
-        [outerHTML]: {
-          ...details,
-          elements: [...details.elements, element],
-        },
-      }
-    }, {} as ElementDetailMap)
+  elements: { [key: string]: Array<Element> } = {}
+  stylesheetElements: Array<Element> = []
+  trackedElemements: Array<Element> = []
```

Comment:
Should this property declaration read `trackedElements` instead of `trackedElemements`?

Reply:
Great example of how one can look at code for days, comb over it for that perfect PR, and miss the simplest errors.